cmd/go-ycsb/aerospike.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+import ( // Register Aerospike database
+	_ "github.com/pingcap/go-ycsb/db/aerospike"
```

Comment:
you can move this to the main.go.

Mostly we will put all registers together except for some which need independent building flags like FoundationDB.